### DISCUSSION TITLE
Feature request: more uses for multipleHits

### DISCUSSION THREAD

**Discussion by eduardoHoefel** [brookhong/Surfingkeys#1497] [Ideas]
I just installed SufingKeys and it is awesome. I really like the level of customization available as well as the premade functions. I saw that for clicking multiple links there is an extra option `multipleHits`. I was wondering if that same option could be used for any click, not only links.

Example:
```
mapkey('c', '', function() {
  Hints.create("", Hints.dispatchMouseClick, {tabbed: false, multipleHits: true});
});
```

With that it would be possible to click on buttons without pressing `c` every time. However when `multipleHits` is used the code interprets that is a situation of opening tabs. Also, it ignores the selectors chosen.

**Comment by ArturGajowy** [comment]
I would love this as well, to the point I'd map it under 'f' :)
